website/app/router.js

```diff
@@ -0,0 +1,13 @@
+import EmberRouter from '@ember/routing/router';
+import config from 'website/config/environment';
+
+export default class Router extends EmberRouter {
+  location = config.locationType;
+  rootURL = config.rootURL;
+}
+
+Router.map(function () {
+  this.route('engineering');
+  this.route('design');
+  this.route('percy-test');
```

Comment:
I am not 100% convinced this should live here, at this point. Since this is actually a test page, maybe (this yes!) should live under the "dummy/test" app? it can be a simple page, with zero styling.
Another VRT test page could be a dynamic page that can be used to test every icon (but in this case the cost of Percy would skyrocket).
Not a blocker, just some ideas in case you want to discuss it.

Reply:
I agree and I struggled with this question as well. Initially I left `percy-test` behind because it _is_ a test for the ember addon itself, but because it has so much tailwind styling and I removed tailwind from `ember-flight-icons` I ended up migrating it over to the website. I could put it back but it seems like it needs _some_ styling since at least some of the tests rely on testing color behavior/etc.

Reply:
@Dhaulagiri if you move it back, I can simplify the page and have it styled without tailwind, it's not complex at all (I'll do it in this PR).

Reply:
I should be able to get it done this afternoon. One nuance we'll have to sort out with this then is configuring multiple percy/vercel setups to deploy/test both `ember-flight-icons` and `website`.

Reply:
✅

Reply:
@Dhaulagiri FYI just pushed changes to the "percy-test" pages in the dummy app, in which I have removed every custom class or Tailwind class from the page, and used just inline styles.